jobs/scripts/i18n-find-untranslated.sh

```diff
@@ -1,7 +1,13 @@
 #!/bin/sh
 set -x
 
-pip install -r requirements.txt
+PYTHON_VERSION="{py}"
+
+uv python install ${{PYTHON_VERSION}}
+uv venv -p ${{PYTHON_VERSION}}
```

Comment:
and if you set the env var `UV_PYTHON="{py}` then you can skip the `-p ...` here  too